src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/MinimumOSAttribute.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+    // Records the minimum platform that is required in order to the marked thing.
```

Comment:
I would convert these comments over to the `///` variety so that our doc generator tools will pick them up.

Reply:
Just pushed the change to convert to xml doc https://github.com/dotnet/runtime/pull/38604/commits/74751693b8e0d24f4c22b141b36b272df8908abc